admin_manual/release_notes.rst

```diff
@@ -141,6 +148,11 @@ Updated minimum supported browser versions
 Users with outdated browsers might get warnings. 
 See `the list of supported browser versions`_.
 
+Known issues
+~~~~~~~~~~~~
+
+- When using application passwords, `log entries related to "Login Failed" will appear <https://github.com/owncloud/core/issues/30157>`_, please upgrade to 10.0.7 and check the fix mentionned in its release notes.
```

Comment:
s/mentionned/mentioned

Reply:
I always make the mistake to add a spare "n" just in case the other one falls off...